### PR TITLE
Add note

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
This PR includes 

- User can add a new note 
- Firestore setup 
- Some bug fix

### CODE REVIEW

File: src/context/data-context.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+const DataContextProvider = ({ children }: DataContextProviderProps) => {
+  const [notes, setNotes] = useState<NoteType[]>([]);
+  const [labels, setLabels] = useState<string[]>([]);
+  const [isLoading, setIsloading] = useState(false);
+  const uid = localStorage.getItem("userId");
+
+  useEffect(() => {
+    getNotes();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (chanchal-panpaliya) [inline src/context/data-context.tsx:29]: remove comment

**Review by chanchal-panpaliya — COMMENTED** [review]
1) code looks clean 
2) proper naming 
3) good UI
4) resolved below issues (if i add long text)
image

**Review by GuptaEesh — COMMENTED** [review]
- Good folder structure.
- Excited to see typescript usage.
- good to merge

File: src/context/data-context.tsx (hunk 1 above)
Comment (junaidshaikh-js) [reply src/context/data-context.tsx:29]: The comment is to disable the ESLint rules.

**Comment by junaidshaikh-js** [comment]
> 1. code looks clean
> 2. proper naming
> 3. good UI
> 4. resolved below issues (if i add long text)
>    image

Solved, thank you!

**Pull request merged**